### PR TITLE
Hoc2022 - NASA skinny banner on Explore, Play, Create landing page

### PR DESCRIPTION
**Pull request by kelbyhawn**
Add a skinny banner to https://code.org/hourofcode2022 promoting a new NASA careers in CS video.
- This is acting as a "coming soon" banner — the button will get added the week before CSEdWeek (starts Dec 5)

**Relevant info:** [Asana link](https://app.asana.com/0/1202764401306810/1203316858646111/f)

----

### Desktop
Desktop

### Tablet
Tablet

### Mobile
Mobile

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/css/hoc2022-landing-page.scss [hunk 1]
```diff
@@ -145,37 +145,6 @@ header {
 
 main {
 
-  // NASA skinny banner
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/css/hoc2022-landing-page.scss:148]: I moved all of this sass code to a separate vanilla css file ([/css/hoc2022-nasa-skinny-banner.css](https://github.com/code-dot-org/code-dot-org/blob/52b92807c28d327b376e7cda68a0029d3a57d0c1/pegasus/sites.v3/code.org/public/css/hoc2022-nasa-skinny-banner.css) or see below) so we can more easily update this in the coming weeks through Dropbox.

File: pegasus/sites.v3/code.org/public/hourofcode2022.haml [hunk 2]
```diff
@@ -36,14 +37,14 @@ theme: responsive
...
-      Copy from NASA goes here
-    %a{href: "#", target: "_blank", rel: "noopener noreferrer"}
-      %button
-        Learn more
+    .wrapper
+      %h1=I18n.t(:hoc2022_nasa_banner_heading)
+      %p=I18n.t(:hoc2022_nasa_banner_description)
+    -# Hiding this until we get the link from NASA the week before Dec 5
+    -# %a{href: "#", target: "_blank", rel: "noopener noreferrer"}
+      %button=I18n.t(:homepage_banner_text_link)
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/public/hourofcode2022.haml:47]: We're hiding the button until NASA sends us the link the week before CSEdWeek (which starts Dec 5). Also, this button will say "Learn more".

File: pegasus/sites.v3/code.org/public/css/hoc2022-landing-page.scss (hunk 1 above)
Comment (bencodeorg) [reply pegasus/sites.v3/code.org/public/css/hoc2022-landing-page.scss:148]: Good call!

File: pegasus/sites.v3/code.org/public/hourofcode2022.haml [hunk 3]
```diff
@@ -36,14 +37,14 @@ theme: responsive
   %img.right{src: "/images/hoc2022-landing-page-banner-right.svg", alt: I18n.t(:hoc2022_landing_page_banner_alt_right)}
 
 %main
-  -# Hiding the NASA banner until post-launch due to mktg reasons
-  -# %aside
+  %aside.nasa-skinny-banner
```
Comment (bencodeorg) [inline pegasus/sites.v3/code.org/public/hourofcode2022.haml:40]: TIL what `<aside>` is :)

**Review by bencodeorg — APPROVED** [review]

**Comment by kelbyhawn** [comment]
Thanks for the quick review @bencodeorg! 🙏

**Pull request merged**